File: cpp/10.cpp
#include <bits/stdc++.h>
using namespace std;

#define ul unsigned long long int

bool isPrime(int num) {
  for (int i = 2; i <= sqrt(num); i++)
    if (num % i == 0) return false;

  return true;
}

int main() {
  ul sum;

  for (int i = 2; i < 2000000; i++)
    if (isPrime(i)) sum += i;

  printf("%lld\n", sum);
}

File: py/10.py
from math import sqrt


def is_prime(num):
    for i in range(2, int(sqrt(num)) + 1):
        if num % i == 0:
            return False

    return True


summation = 0
for i in range(2, 2000000):
    if is_prime(i):
        summation += i

print(summation)
